frontend/src/components/PrescreeningForm.tsx

```diff
@@ -0,0 +1,227 @@
+            name="question1"
+            checked={question1Value === "op13"}
+            onChange={() => {
+              setQuestion1Value("op13");
+            }}
+          />
+
+          <label htmlFor="op13" className={styles.label}>
+            <p>
+              I do not have 5 years of Diversified Design Experience or have 40+ Core Units and
+              <strong> will not complete </strong> at least one of these two requirements{" "}
+              <strong> within a year </strong> from today.
+            </p>
+          </label>
+        </div>
+
+        {/* Question 2 */}
+        <div>
+          <p className={styles.question}>
+            2. If you have passed any of the following qualifying National Interior Design Exams,
+            please indicate so
+            <ul className={styles.ulprescreening}>
+              <li>ARE</li>
+              <li>CASP</li>
+              <li>LEED - AP (+)</li>
+              <li>NCBDC</li>
+              <li>NCIDQ</li>
+              <li>NKBA - CKBD (+)</li>
+            </ul>
+          </p>
+
+          <br></br>
+
+          <input
+            type="radio"
+            id="op21"
+            name="question2"
+            checked={question2Value === "op21"}
+            onChange={() => {
+              setQuestion2Value("op21");
+            }}
+          />
+
+          <label htmlFor="op21" className={styles.label}>
+            Yes
+          </label>
+
+          <br></br>
+
+          <input
+            type="radio"
+            id="op22"
+            name="question2"
+            checked={question2Value === "op22"}
+            onChange={() => {
+              setQuestion2Value("op22");
+            }}
+          />
+
+          <label htmlFor="op22" className={styles.label}>
+            No
+          </label>
+        </div>
+
+        {/* Question 3 */}
+        <div>
+          <p className={styles.question}>3. Do you mainly practice commercial design?</p>
+
+          <br></br>
+
+          <input
+            type="radio"
+            id="op31"
+            name="question3"
+            checked={question3Value === "op31"}
+            onChange={() => {
+              setQuestion3Value("op31");
+            }}
+          />
+
+          <label htmlFor="op31" className={styles.label}>
+            Yes
+          </label>
+
+          <br></br>
+
+          <input
+            type="radio"
+            id="op32"
+            name="question3"
+            checked={question3Value === "op32"}
+            onChange={() => {
+              setQuestion3Value("op32");
+            }}
+          />
+          <label htmlFor="op32" className={styles.label}>
+            No
+          </label>
+        </div>
+
+        <div className={styles.centeredContainer}>
+          <button className={buttonStyle.button} type="submit" form="form1" value="Submit">
+            Submit
+          </button>
+        </div>
+      </form>
+    </Page>
+  );
+}
+
+export default PrescreeningForm;
```

Comment:
We don't need this line here. We exported the function before and the convention in this project is to not use export default